components/card/CollapsableCard.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components/ui/Accordion"
+import React from "react"
+import { StyledCardProps } from "@/components/card/StyledCard"
+import { cn } from "@/lib/utils"
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/Card"
+import Icon from "@/components/ui/RenderIcon"
+import { CardVariants } from "@/components/card/variants"
+
+export const CollapsableCard: React.FC<StyledCardProps> = ({
+  iconName,
+  isDisabled = false,
+  title,
+  className,
+  size,
+  children,
+  footer,
+}) => {
+  const IconComponent = iconName ? Icon : null
+  return (
+    <Card
+      className={cn(
+        CardVariants({
+          size,
+          className,
+        }),
+        isDisabled ? "opacity-30 grayscale" : "",
+        "flex flex-col"
+      )}
+    >
+      <Accordion type="single" collapsible>
+        <AccordionItem className="border-none" value="item-1">
+          {title && (
+            <CardHeader className="flex-shrink-0">
+              <CardTitle className="flex items-center justify-center gap-4 border-b border-gray-300 text-center">
```

Comment:
i'm not a fan of the bottom border. also, could the text and arrow behave the same as the navbar but maybe keppel instead of sunglow on hover? and the arrow flips on hover/when card is open